src/pages/index.js

```diff
@@ -98,4 +99,30 @@ const Index = () => {
   );
 };
 
+export const getStaticProps = async () => {
+  const fallbackUserCount = 200000;
+
+  try {
+    const r = await fetch(
```

Comment:
I should be less lazy and check this myself but does this Fetch implementation throw on 4/5xx responses? I know the browser Fetch API only throws on network errors. 

I can see that it'd default if a situation occurred where it didn't throw and the `approximate_member_count` was still falsy but I thought I'd ask.

Reply:
Yep! Anything that isn't a 200 will throw to the catch block

Reply:
Awesome, thanks!